cluster-scope/overlays/nerc-ocp-obs/kustomization.yaml

```diff
@@ -53,13 +55,6 @@ patches:
             - ocp-on-nerc/nerc-ops
             - ocp-on-nerc/nerc-logs-metrics
             - ocp-on-nerc/nerc-obs-admins
-- target:
-    kind: ExternalSecret
-    name: oauths-clientsecret-nerc
-  patch: |
-    - op: replace
-      path: /spec/data/0/remoteRef/key
-      value: nerc/nerc-ocp-obs/openshift-config/oauths-clientsecret-nerc
```

Comment:
Why are we removing this patch?

Reply:
@larsks I thought they are only for keycloak.mss (only in prod right now), and I assumed we don't want this (yet) for the obs, therefore I streamlined it.
But maybe I am wrong in one or both of these thoughts?

Reply:
Thanks, that makes sense.

Would you update the commit message to include this information?

Reply:
thanks, will do.